optaplanner-persistence/optaplanner-persistence-jackson/pom.xml

```diff
@@ -58,6 +58,17 @@
       <groupId>com.fasterxml.jackson.core</groupId>
       <artifactId>jackson-databind</artifactId>
     </dependency>
+    <!-- Testing -->
+      <dependency>
+      <groupId>org.powermock</groupId>
+      <artifactId>powermock-module-junit4</artifactId>
```

Comment:
We standarize on Mockito, so we don't want to use powermock, as that will increase complexity (harder for developers to read code as they now have to know 2 mocking frameworks) and decrease reuse opportunity (see PlannerAssert.java which already has Mockito helper methods specifically for optaplanner interfaces).

Reply:
I removed Powermock as I found another solution when solving a different a different issue. However, Powermock shares Mockito API: All Mockito methods are defined in Powermock Mockito API (So you only need to know one mocking framework). All Powermock adds is the ability to mock constructors, final classes and static methods. Due to poor documentation on how to configure jackson in unit tests, I used it simulate ObjectMapper injection.